### PR TITLE
SB-22862 - Collection CSV API end documentation

### PR DESCRIPTION
**Pull request by Jayaprakash8887**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @Jayaprakash8887,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://759.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://759.qa.docs.sunbird.org/3.9.0/apis/collectioncsvapi// <br /> - http://759.qa.docs.sunbird.org/3.9.0/apis/collectioncsvapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Jayaprakash8887,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://759.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://759.qa.docs.sunbird.org/3.9.0/apis/collectioncsvapi// <br /> - http://759.qa.docs.sunbird.org/3.9.0/apis/collectioncsvapi// <br />  <br/> **Modified sections are -** <br />

File: apis/collectioncsvapi/collectioncsvapi.yaml [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+          in: header
+          description: Access token of registered user performing given API request.
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '{{api-key}}'
+        - name: collection-id
+          in: path
+          description: 'Collection id to upload. It should be unique and same as csv file identifier'
```
Comment (Sunitashelar) [inline apis/collectioncsvapi/collectioncsvapi.yaml:81]: @Jayaprakash8887 globally modify 'id' to ID

File: apis/collectioncsvapi/collectioncsvapi.yaml [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+                - fileUrl
+              type: object
+              properties:
+                fileUrl:
+                  type: string
+                  description: 'File which user want to upload'
+        required: false
+      responses:
+        '200':
+          description: 'SUCCESS. The "TOC Upload" operation was successful!'
```
Comment (Sunitashelar) [inline apis/collectioncsvapi/collectioncsvapi.yaml:101]: @Jayaprakash8887 modify the message to ' SUCCESS. The TOC was uploaded successfully.'

File: apis/collectioncsvapi/collectioncsvapi.yaml [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+      deprecated: false
+      security:
+        - bearer: []
+  '/collection/v1/toc/download/{collection-id}':
+    get:
+      tags:
+        - TOC APIs
+      summary: TOC Download
+      description: |-
+        This API is associated with download collection in csv format.
```
Comment (Sunitashelar) [inline apis/collectioncsvapi/collectioncsvapi.yaml:183]: @Jayaprakash8887 can this description be changed to ' This API is used to download the collection in .csv format.'?

File: apis/collectioncsvapi/collectioncsvapi.yaml [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+
+        The endpoint for TOC download is /download
+        The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+      operationId: TOCDownload
+      parameters:
+        - name: Content-Type
+          in: header
+          description: "The Content Type entity is the media type of the resource. Possible\
```
Comment (Sunitashelar) [inline apis/collectioncsvapi/collectioncsvapi.yaml:191]: @Jayaprakash8887 please refer to the earlier comment on the use of \ to mark line breaks. this is a global comment.

File: apis/collectioncsvapi/collectioncsvapi.yaml [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+        - name: collection-id
+          in: path
+          description: 'Collection id which user want to download'
+          required: true
+          style: simple
+          schema:
+            type: string
+      responses:
+        '200':
+          description: 'SUCCESS. The "TOC Download" operation was successful!'
```
Comment (Sunitashelar) [inline apis/collectioncsvapi/collectioncsvapi.yaml:223]: @Jayaprakash8887 refer to earlier success message sentence framing.

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

**Comment by Jayaprakash8887** [comment]
Created new pull request against 4.0. #764

**Pull request closed without merging**